src/x86/cpuid.c

```diff
@@ -767,6 +771,32 @@ struct topology* get_topology_info(struct cpuInfo* cpu, struct cache* cach, int
   return topo;
 }
 
+struct cache* get_cache_info_transmeta(struct cache* cach) {
```

Comment:
Why can't we use `get_cache_info_general`. It does not work?

Reply:
It failed to retrieve the cache size with that. I have eventually hardcoded the cache size to this CPU model.